### ISSUE TITLE
This doesn't support mutations?

### ISSUE THREAD

**Issue by jackyzhen** [kadirahq/graphqlify#3]
Maybe I'm not understanding how to work this but I can't build a mutation with this library. Is there an example of a mutation request?

**Comment by thani-sh** [comment]
You can do something like this:

``` js
const mutation = 'mutation ' + graphqlify({ setFoo: { params{value: 100} } })
// mutation {setFoo(value:100)}
```

I guess it'll be easier if we implement helper functions for these. What do you think?

``` js
// this is not available right now
const query = graphqlify.query({})
const mutation = graphqlify.mutation({})
```

**edit:** fix the example javascript code (first method)

**Comment by thani-sh** [comment]
Fixed with #6 
Released `v1.1.0`
